examples/test.cpp

```diff
@@ -1,6 +1,43 @@
-#include "../include/featurelib.h"
+#include "../include/feature_lib.h"
+#include <iostream>
+#include <opencv2/core.hpp>
+#include <opencv2/imgcodecs.hpp>
 
-int main() {
-  featurelib instance;
+std::vector<uint8_t> matToVector(const cv::Mat &mat) {
+  std::vector<uint8_t> result;
+  cv::Mat cont = mat.isContinuous() ? mat : mat.clone();  // make sure the Mat is continuous
+  cont.reshape(1, 1).copyTo(result);  // copy from a 1D Mat
+  return result;
+}
+//Please inter only filepath to command line arguments
```

Comment:
Add a blank line between `matToVector` and this doc

Reply:
done